client/verta/verta/registry/errors.py

```diff
@@ -0,0 +1,18 @@
+class CustomModelError(Exception):
```

Comment:
How would you feel about naming it `ModelError` rather than `CustomModelError`? The "Custom" feels a little bit on the nose to me.